Saving trajectories

DjoleMNE opened the pull request:
Package for saving trajectories created. It takes as parameters: names of start and end configurations. Based on these parameters it generates/plans trajectories using moveit. Next, it stores these plans under (by default) mcr_moveit_client. However, the desired destination of these plans can be changed in the launch file.  

Pre-planned trajectories saved using yaml files in config folder under mir_moveit_client package.

Under mcr_arm_motions trajectory executor is created. It executes pre-planned trajectories, which are saved using aforementioned package. It is a separate software component, which can be used testing feasibility of saved trajectories.

However, this type of trajectory execution is implemented under move_arm class in manipulation states (mas_industrial repository), in order to be used by state machine directly.

DjoleMNE committed:
[Trajectory saving] Package for trajectory saving added

Package for saving trajectories created. It takes as parameters: names of start and end configurations. Based on these parameters it generates/plans trajectories using moveit. Next, it stores these plans under (by default) mcr_moveit_client. However, the desired destination of these plans can be changed in the launch file.

DjoleMNE committed:
[Trajectory  saving] Trajectories saved via yaml files  in config folder

Pre-planned trajectories saved using yaml files  in config folder under mcr_moveit_client package

DjoleMNE committed:
[Trajectory saving] Trajectory executor created under mcr_arm_motions 

Under mcr_arm_motions trajectory executor is created. It executes pre-planned trajectories, which are saved using aforementioned package. It is a separate software component, which can be used testing feasibility of saved trajectories.

DjoleMNE committed:
[Trajectory saving] Plans for place_poses saved in yaml files

DjoleMNE committed:
[Trajectory saving] New plans saved for unstage server motions

DjoleMNE committed:
[Trajectory saving] New set of trajectories saved. Required for stage SM

sthoduka commented:
@DjoleMNE the saved trajectories should probably be in mir_moveit_client since it is only for the youbot

DjoleMNE commented:
Noted! It will be changed.

DjoleMNE committed:
[Trajectory saving] Specified path to saved trajectory files changed

DjoleMNE committed:
[Trajectory saving] Saved trajectory files removed from mas_common repo.

Trajectory files moved to unmerged_packages repository under mir_moveit_client package

DjoleMNE commented:
@sthoduka  Done! and tested on real robot with all 3 SM
